x-pack/plugins/cases/public/common/mock/connectors.ts

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
Moved it from `x-pack/plugins/cases/public/containers/configure/mock.ts` to a new common file.